linux/ProcessField.h

```diff
@@ -44,8 +44,10 @@ in the source distribution for its full text.
    M_PSSWP = 121,                \
    CTXT = 122,                   \
    SECATTR = 123,                \
+   AUTOGROUP = 127,              \
+   AUTOGROUP_NICE = 128,         \
                                  \
-   DUMMY_BUMP_FIELD = CWD,       \
+   DUMMY_BUMP_FIELD = 128,       \
```

Comment:
Can be dropped, was only there to ensure the last ID defined here was the overall highest ID available.